pkg/agent/agentclient/clientset.go

```diff
@@ -128,30 +128,48 @@ func (cs *ClientSet) newAgentClient() (*AgentClient, error) {
 	return newAgentClient(cs.address, cs.agentID, cs, cs.dialOption)
 }
 
+func (cs *ClientSet) resetBackoff() *wait.Backoff {
+	return &wait.Backoff{
+		Steps:    3,
+		Jitter:   0.1,
+		Factor:   1.5,
+		Duration: cs.syncInterval,
+	}
+}
+
 // sync makes sure that #clients >= #proxy servers
 func (cs *ClientSet) sync() {
-	jitter := float64(0.2)
+	backoff := cs.resetBackoff()
+	var duration time.Duration
 	for {
-		if cs.serverCount != 0 {
-			sleep := cs.syncInterval + time.Duration(rand.Float64()*jitter*float64(cs.syncInterval))
-			time.Sleep(sleep)
-		}
-		if cs.serverCount == 0 || cs.ClientsCount() < cs.serverCount {
-			c, err := cs.newAgentClient()
-			if err != nil {
-				klog.Error(err)
-				continue
-			}
-			cs.serverCount = c.stream.serverCount
-			if err := cs.AddClient(c.stream.serverID, c); err != nil {
-				klog.Infof("closing connection: %v", err)
-				c.Close()
-				continue
-			}
-			klog.Infof("sync added client connecting to proxy server %s", c.stream.serverID)
-			go c.Serve()
+		if err := cs.syncOnce(); err != nil {
+			klog.Error(err)
+			duration = backoff.Step()
+		} else {
+			backoff = cs.resetBackoff()
+			duration = wait.Jitter(backoff.Duration, backoff.Jitter)
```

Comment:
based on Jitter documentation, we can set second parameter as 0.0, so default value will be selected

Reply:
I think using the value defined in this file is more readable than using the default value.